### ISSUE TITLE
global_tags does not seem to work

### ISSUE THREAD

**Issue by mcuelenaere** [DataDog/dd-trace-py#763]
The following piece of code does not set the global tag `env` to `'baz'`:

```python
tracer = Tracer('foo', config={
    'agent_hostname': 'bar',
    'agent_port': 8126,
    'enabled': True,
    'global_tags': {
        'env: 'baz'
    }
})
```

This only works if I add the following hack:
```python
tracer._dd_tracer.set_tags({'env': 'baz'})
```

**Comment by Kyle-Verhoog** [comment]
Hi @mcuelenaere! I assume that you're using our beta opentracer (thanks for trying it out!). I did a quick look and it seems like we're missing the functionality for passing the global tags from the opentracer to the underlying datadog tracer.

It's a straightforward fix, we'll get it in our next release.

Thanks for the report!

**Comment by brettlangdon** [comment]
This issue has been resolved by #764.

@mcuelenaere if you are still having issues, please let us know and re-open this issue.
